text/0024-schema-syntax.md

```diff
@@ -458,3 +460,238 @@ Mitigating this problem: The Rust internals already parse the JSON schema to a d
 One alternative would be to _replace_ the current JSON-based sytnax with the one in this proposal. This proposal would avoid the "cognitive burden" drawback mentioned above, but would be a disruptive, backward-incompatible change, and would lose the JSON format's benefits of existing tooling and easier programmatic schema construction.
 
 Another alternative would be to adopt a [Yaml](https://en.wikipedia.org/wiki/YAML)-based syntax. This approach would meet our goals of greater information density and support for comments, and it would come with some existing tooling (such as IDE extensions). A downside of Yaml is that it provides _more_ than we need, with a lack of conciseness leading to confusing. We could make our own parser for a subset of Yaml we wish to support for schemas, but that may lead to a confusing user experience. Yaml's indentation-sensitive parsing also means that an indentation mistake will be silently accepted, leading to a confusing user experience. Our custom syntax is whitespace-insensitive, and having total control over the grammar means better context for error messages.
+
+
+
+## Appendix A - AppliesTo Conversion Table
+
+The relationship of `appliesTo` clauses between JSON schemas and this new syntax are non-obvious when unspecified entities or actions with no targets are involved. This appendix details a conversion table for the purposes of specification.
+
+
+## JSON → Schema Syntax
```

Comment:
This heading is probably unnecessary as there is no Schema -> JSON section (that I saw on first glance)?  Also, it's the same heading weight as the Appendix A label, which is a little weird